libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/files/ConnectUriBuilderTest.java

```diff
@@ -35,8 +35,14 @@
 
 public class ConnectUriBuilderTest extends TestCase {
 
-    private final String connectPath = "/services/data/" + ApiVersionStrings.getVersionNumber(SalesforceSDKManager.getInstance().getAppContext()) + "/chatter/";
```

Comment:
Not calling getInstance() (which throws an exception if app has not yet been initialized) from a member initializer.